frontend/test/metabase/scenarios/dashboard/parameters.cy.spec.js

```diff
@@ -125,13 +125,10 @@ describe("scenarios > dashboard > parameters", () => {
     cy.log(
       "**URL is updated correctly with the given parameter at this point**",
     );
-    cy.url().should("include", "category=Gizmo");
 
-    // Remove filter and save dashboard
-    cy.get(".Icon-pencil").click();
-    cy.get(".Dashboard .Icon-gear").click();
-    cy.findByText("Remove").click();
-    cy.findByText("Save").click();
+    cy.url().should("include", "category=Gizmo");
+    cy.get(".Icon-close").click();
```

Comment:
I don't think this test is reproducing the issue described in #10829. For that, we should delete the name of the filter.

When the bug existed, the parameter would suddenly disappear. Now I think it should get a new name.

Reply:
Clicking "X" **is** deleting the name of the filter :)
![ ](https://p58.f1.n0.cdn.getcloudapp.com/items/7KubmYPR/Screen%20Recording%202020-12-07%20at%2003.43.11%20PM.gif)

Reply:
That clears the value in the filter. The name of the filter there is "Category".

When the bug was filed you edited the filter name at the top of the widget, but now that happens in the sidebar.